subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -98,6 +97,26 @@ static const char *const job_status_strings[] = {
 #define JOB_STATUS_STRING_COUNT (sizeof(job_status_strings) / \
 				 sizeof(*job_status_strings))
 
+/* Define a string representing the network protocol we are using. */
+#if defined(CONFIG_NRF_CLOUD_MQTT)
+#define NRF_CLOUD_JSON_VAL_CFGD_PROTO_VAL NRF_CLOUD_JSON_VAL_PROTO_MQTT
+#elif defined(CONFIG_NRF_CLOUD_REST)
+#define NRF_CLOUD_JSON_VAL_CFGD_PROTO_VAL NRF_CLOUD_JSON_VAL_PROTO_REST
+#elif defined(CONFIG_NRF_CLOUD_COAP)
+#define NRF_CLOUD_JSON_VAL_CFGD_PROTO_VAL NRF_CLOUD_JSON_VAL_PROTO_COAP
+#else
+#define NRF_CLOUD_JSON_VAL_CFGD_PROTO_VAL "Unknown"
+#endif
+
+/* Define a string represention what connection method we are using. */
+#if defined(CONFIG_NRF_MODEM_LIB)
```

Comment:
can `CONFIG_WIFI`  and `CONFIG_NRF_MODEM_LIB` both be enabled?
i think that is a valid use case, but not sure if it works now?

Reply:
Yes, they can be both enabled, but only for Wi-Fi scanning for location services, with LTE being the only fully operating network stack.

The Wi-Fi stack plus Mbed-TLS plus Zephyr socket and UDP stack are all too large to fit on the nrf916x with the modem and modemlib.

Reply:
Yes, but I do not think there is a way to determine it at runtime yet, correct?

Reply:
I haven't looked into it deeply, there are some potential avenues, but I'm not sure whether any of them would even be the right choice -- it's just something I think should be on our radar

Reply:
https://nordicsemi.atlassian.net/browse/IRIS-6842